common/app/routes/challenges/components/Show.jsx

```diff
@@ -69,11 +76,13 @@ export class Challenges extends PureComponent {
     params: PropTypes.object.isRequired,
     areChallengesLoaded: PropTypes.bool,
     resetUi: PropTypes.func.isRequired,
-    updateTitle: PropTypes.func.isRequired
+    updateTitle: PropTypes.func.isRequired,
+    challengeTitle: PropTypes.string
```

Comment:
We shouldn't need two titles here, or in any other component.

We should stick to using just `title` instead of qualifying it as challengeTitle.